Algorithm/QCAlgorithm.Framework.cs

```diff
@@ -264,10 +253,26 @@ public void SetAlpha(IAlphaModel alpha)
             Alpha = alpha;
         }
 
+        /// <summary>
+        /// Adds a new alpha model
+        /// </summary>
+        /// <param name="alpha">Model that generates alpha to add</param>
+        public void AddAlphaModel(IAlphaModel alpha)
+        {
+            if (Alpha.GetType() != typeof(NullAlphaModel))
+            {
+                Alpha = new CompositeAlphaModel(Alpha, alpha);
```

Comment:
Please add `Add` method to CompositeAlphaModel to prevent nesting

Reply:
Added the new method + a Py/C# regression test combo :)